packages/frontend-shared/src/gql-components/OpenFileInIDE.vue

```diff
@@ -38,7 +38,7 @@ const props = defineProps<{
   filePath: string
 }>()
 
-const query = useQuery({ query: OpenFileInIdeDocument, requestPolicy: 'network-only' })
+const query = useQuery({ query: OpenFileInIdeDocument, requestPolicy: 'cache-and-network' })
```

Comment:
Was this necessary? I think it might be `network-only` for a reason (could be wrong).